src/v3/src/components/Widget/index.tsx

```diff
@@ -223,19 +222,12 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
     /**
      * polling transaction name can sometimes be the same as the current transaction's name
      * i.e. unlocking an account and immeidiately being challenged with a different authenticator
-     * when that is the case, we need to compare inputs to determine if it is the same transaction
-     * lastly, inputs could also be the same (i.e. email otp to password) in that case we finally
-     * check the authenticator key to determine if they are the same
-     * (as you shouldnt be challenged with the same authenticator)
+     * when that is the case, we need to check the authenticator key to
+     * determine if they are the same (as you shouldn't be challenged with the same authenticator)
     */
-    const pollingTxAndCurrentTxInputsDiffer = !isEqual(
-      pollingTransaction.nextStep?.inputs,
-      idxTransaction.nextStep?.inputs,
-    );
     const pollingTxAuthKey = getAuthenticatorKey(pollingTransaction);
     const currentTxAuthKey = getAuthenticatorKey(idxTransaction);
     if (pollingTransaction.nextStep?.name !== idxTransaction.nextStep?.name
-        || pollingTxAndCurrentTxInputsDiffer
```

Comment:
I had to remove this because when there are field level errors, the `isEqual` function always returns false (even when it should be true. In this specific scenario, there isnt a need to check the inputs since we are also checking authenticator keys. So I removed this to check the name and authenticator keys only.